### ISSUE TITLE
error when using a uri with a param

### ISSUE THREAD

**Issue by vitosamson** [feignjs/feignjs#5]
uri-templates is throwing an error when I try to make a request with a url param:

```
Uncaught TypeError: valueFunction is not a function
  subFunction @ uri-templates.js:97
  fill @ uri-templates.js:336
  Wrapper.getProcessedUrl @ feign.web.js:177
  Wrapper.executeRequest @ feign.web.js:200
  feign.promiseProxyFactory @ feign.web.js:22
```

Here's the code I'm testing with:

```
var api = {
  me: 'GET /users/me',
  getAsset: 'GET /assets/{id}'
};

var client = feign.builder(true)
            .client(new FeignJquery({
              debug: true,
              headers: {
                Authorization: 'Bearer ' + window.localStorage.getItem('token')
              }
            }))
            .decoder(new feign.JsonDecoder())
            .encoder(new feign.JsonEncoder())
            .requestInterceptor({
              apply: function(req) {
                console.log(req); // this never gets called for api.getAsset
              }
            })
            .target(api, 'http://localhost:3000/v1');

// this works fine
client.me().then(function(res) {
  console.log(res);
}, function(err) {
  console.log(err);
});

// this causes the error
client.getAsset({id: 6}).then(function(res) {
  console.log(res);
});
```

feignjs@0.0.4 (installed from npm)
feignjs-jquery@0.0.4 (installed from npm)
args-js#0.10.7 (installed from bower)
uri-templates#0.1.5 (installed from bower)

**Comment by warmuuh** [comment]
try installing feignjs and feignjs-jquery via bower because it fetches necessary dependencies automatically,e.g. uri-template ver 0.1.7...
i am not sure, if this is the issue though and i will have a look later on

**Comment by warmuuh** [comment]
i see... uri-templates is not available in version 0.1.7 (because there is no tag in the repo) so i adjusted the sample to use 0.1.7 (via <giturl>#master bower-dependency), now it works... see samples/bower.json
